group/lc/main.tf

```diff
@@ -3,13 +3,13 @@
 ## Creates security group
 resource "aws_security_group" "sg_asg" {
   description = "${var.stack_item_fullname} security group"
-  name_prefix = "${var.stack_item_label}-asg-"
+  name_prefix = "${var.stack_item_label}${var.sg_name_suffix}"
```

Comment:
If I understand correctly that would similar to the logic used here: [template_file name](https://github.com/unifio/terraform-aws-asg/blob/master/group/main.tf#L17)